### PR TITLE
New Rule: Google impersonation Fake sign-in warning image lure

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

**Comment by morriscode** [comment]
testing looks safe, this is pretty specific should be gtg

**Review by aidenmitchell — APPROVED** [review]

File: detection-rules/link_google_fake_sign_in_image_lure.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: "Link: Google impersonation fake sign-in warning"
```
Comment (jkamdjou) [inline detection-rules/link_google_fake_sign_in_image_lure.yml:1]:
```suggestion
name: "Brand impersonation: Google fake sign-in warning"
```

File: detection-rules/link_google_fake_sign_in_image_lure.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+description: |
+  Detects emails with image attachments containing fake Google sign-in warnings and ensures that the links in the email body do not lead to Google sites.
```
Comment (jkamdjou) [inline detection-rules/link_google_fake_sign_in_image_lure.yml:3]:
```suggestion
  Detects messages with image attachments containing fake Google sign-in warnings with no links leading to Google sites.
```

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**